tox.ini

```diff
@@ -0,0 +1,86 @@
+# This file is managed by bootstack-charms-spec and should not be modified
```

Comment:
This is template for reactive charms, could we create one for `ops` and than use it here?
- wrong `PYTHONPATH`
- wrong coverage setting in unit tests

Reply:
I've applied the templates created in this [MP](https://code.launchpad.net/~rgildein/bootstack-charms-spec/+git/bootstack-charms-spec/+merge/429914)